finetune/encoding/target_encoders.py

```diff
@@ -229,21 +249,338 @@ def transform(self, out, labels):
                 if overlap:
                     if not agree:
                         raise ValueError("Tokens and labels do not align")
-                    if labels_out[i] != pad_idx and self.lookup[label["label"]] != labels_out[i]:
+                    if labels_out[i] != pad_idx and self.lookup[current_label] != labels_out[i]:
                         LOGGER.warning("Overlapping labels were found, consider multilabel_sequence=True")
-                    if label["label"] not in self.lookup:
+                    if current_label not in self.lookup:
                         LOGGER.warning(
                             "Attempting to encode unknown labels : {}, ignoring for now but this will likely not "
-                            "result in desirable behaviour. Available labels are {}".format(label["label"], self.lookup.keys())
+                            "result in desirable behaviour. Available labels are {}".format(current_label, self.lookup.keys())
                         )
                     else:
-                        labels_out[i] = self.lookup[label["label"]]
+                        labels_out[i] = self.lookup[current_label]
```

Comment:
Bit uncomfortable that output space is 2 x 3 x n_classes when bio_tagging and group_tagging is used but I guess if the same classes nearly always begin the group there's no issue